packages/terra-date-picker/src/DatePickerField.jsx

```diff
@@ -0,0 +1,234 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Field from 'terra-form-field';
+import DatePicker from './DatePicker';
+
+
+const propTypes = {
+  /**
+   * The DatePicker identifier. Links the htmlFor of the field to the select identifier.
+   */
+  datePickerId: PropTypes.string.isRequired,
+  /**
+   * Whether the date input should be disabled.
+   */
+  disabled: PropTypes.bool,
+  /**
+   * Error message for when the input is invalid. This will only be displayed if isInvalid is true.
+   */
+  error: PropTypes.node,
+  /**
+   * Error Icon to display when the input is invalid.
+   */
+  errorIcon: PropTypes.element,
+  /**
+   * An array of ISO 8601 string representation of the dates to disable in the picker.
+   */
+  excludeDates: PropTypes.arrayOf(PropTypes.string),
+  /**
+   * A function that gets called for each date in the picker to evaluate which date should be disabled.
+   * A return value of true will be enabled and false will be disabled.
+   */
+  filterDate: PropTypes.func,
+  /**
+   * Help element to display with the input.
+   */
+  help: PropTypes.node,
+  /**
+   * Whether or not to hide the required indicator on the label.
+   */
+  hideRequired: PropTypes.bool,
+  /**
+   * Whether or not the field is an inline field.
+   */
+  isInline: PropTypes.bool,
+  /**
+   * Whether or not the field is invalid.
+   */
+  isInvalid: PropTypes.bool,
+  /**
+   * Whether or not the label is visible. Use this props to hide a label while still creating it on the DOM for accessibility.
+   */
+  isLabelHidden: PropTypes.bool,
+  /**
+   * An array of ISO 8601 string representation of the dates to enable in the picker. All Other dates will be disabled.
+   */
+  includeDates: PropTypes.arrayOf(PropTypes.string),
+  /**
+   * Custom input attributes to apply to the date input. Use the name prop to set the name for the input.
+   * Do not set the name in inputAttribute as it will be ignored.
+   */
+  // eslint-disable-next-line react/forbid-prop-types
+  inputAttributes: PropTypes.object,
```

Comment:
Should we use `inputAttrs` to match other form-field wrappers? 

https://github.com/cerner/terra-core/blob/master/packages/terra-form-input/src/InputField.jsx#L46

Reply:
I used `inputAttributes` here because it's the name of the prop on the DatePicker

Reply:
We've used both [inputAttrs](https://github.com/cerner/terra-core/blob/master/packages/terra-form-input/src/InputField.jsx#L46) and [inputAttributes](https://github.com/cerner/terra-core/blob/master/packages/terra-search-field/src/SearchField.jsx#L79) on different components. I like `inputAttrs` but I think consumers aren't really sure what it is used for so I lean towards using `inputAttributes` going forward as it is more explicit what the prop is used for.

Reply:
Here is the original comment on deciding on the name for this prop and decided to go with `inputAttributes`: https://github.com/cerner/terra-core/pull/292#discussion_r114352918